src/prng/isaac_serde.rs

```diff
@@ -0,0 +1,68 @@
+#[cfg(feature="serde-1")]
```

Comment:
This file needs a header. Copy the copyright/licence block and add a one-line `//!` comment please.

Also I think this `cfg` line is redundant since it was also applied to `mod isaac_serde;`.